test/test-blocklist.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+/**
+ * List of tests that should not run in the Angular component test suites. This should
+ * be empty in the components repository, but the file will be overwritten if the framework
+ * repository runs the Angular component test suites against the latest snapshots. This is
+ * helpful because sometimes breaking changes that break individual tests land in the framework
+ * repository. It should be possible to disable these tests until the component repository
+ * migrated the broken tests.
+ */
+export const testBlocklist: {[testName: string]: Object} = {};
```

Comment:
Should this be `BlocklistEntry` like we have in Framework?

Reply:
Good question. I decided not to do that as on the components side we are not interested in notes or error message but rather just in the list of disabled tests.

Happy to change though. I don't feel too strong :smile:

Reply:
Whichever way is fine with me, we can leave it.  Its a temporary piece anyway that we shouldn't need for a long time anyway.